src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        return Colors.values()[new Random().nextInt(Colors.values().length)];
```

Comment:
Introduce `Random` instance as field of the class, no need to create it every time `getRandomColor` called.